app/src/main/java/com/paocu/xviss/activities/BaseActivity.java

```diff
@@ -18,17 +18,45 @@
 import androidx.drawerlayout.widget.DrawerLayout;
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.appcompat.widget.Toolbar;
+import androidx.recyclerview.widget.DividerItemDecoration;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
 import com.paocu.xviss.MainActivity;
 import com.paocu.xviss.R;
 import com.paocu.xviss.TravelListActivity;
+import com.paocu.xviss.activities.adapter.TravelItemAdapter;
+import com.paocu.xviss.model.Travel;
+import com.paocu.xviss.network.RetrofitNetwork;
+import com.paocu.xviss.network.requests.TravelService;
+import com.paocu.xviss.services.TravelLiveService;
 
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import retrofit2.Response;
+
+/**
+ * this activity shows all user travels
+ */
 public class BaseActivity extends AppCompatActivity {
 
     private AppBarConfiguration mAppBarConfiguration;
 
+    private RecyclerView recyclerView;
+
+    private TravelLiveService travelLiveService;
+    private TravelService travelService;
+
+    private final ExecutorService executorService = Executors.newFixedThreadPool( 1 );
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        System.out.println("CREATING");
```

Comment:
Avoid printing to console to get debugging messages. Use a proper logger instead